assets/src/edit-story/components/library/panes/text/textSets/textSet.js

```diff
@@ -114,7 +126,14 @@ function TextSet({ elements, translateY, translateX, ...rest }, ref) {
       onKeyUp={handleKeyboardPageClick}
       {...rest}
     >
-      <TextSetElements isForDisplay elements={elements} />
+      {renderImages ? (
+        <TextSetImg
+          src={`${cdnURL}images/text-sets/${id}.png`}
+          alt={rest['aria-label']}
```

Comment:
```suggestion
          alt={rest['aria-label']}
          crossorigin="anonymous"
```

This is needed because the images are loaded from a different host and we set up cross-origin isolation when video optimization is enabled.